### PR TITLE
moving deltaView to a component, showing selected page for launch

### PR DESCRIPTION
**Pull request by opishcheiko**
- adding simple scrolling to the top, when test selected
- removing unused stuff from Basepage component for better readability, adding comments here and there
- moving deltaView functions  (showing pyramids) into a separate component for readability
- highlighting pagination on the launch page
Screenshot 2020-07-22 at 16 43 11

- some small twigs on the test history (adding a badge for Current Test)
- experimenting with tabs, thought to simplify them a bit
Screenshot 2020-07-22 at 21 18 04

### CODE REVIEW

**Comment by opishcheiko** [comment]
@Jnegrier refactoring here as alwaaays

File: pages/charts.tsx [hunk 1]
```diff
@@ -119,7 +119,7 @@ function Charts(props: Props) {
 // https://nextjs.org/docs/api-reference/data-fetching/getInitialProps
 
 Charts.getInitialProps = async (): Promise<Props> => {
-  const projectReq = await fetch(`http://delta-core.dsch.dev/api/v1/projects`, {
+  const projectReq = await fetch(`${process.env.deltaCore}/api/v1/projects`, {
```
Comment (Jnegrier) [inline pages/charts.tsx:122]: Don't tell me that this was me D: hoooo no
I would have to be more careful next time, a thousand apologies for this 😿 
and a million thanks for fixing it!

Comment (opishcheiko) [reply pages/charts.tsx:122]: of course it was me, pfff 🙈 
We say, better later than never :) Didn't notice i did it

**Comment by opishcheiko** [comment]
@Jnegrier what are those tests actually doing?
https://github.com/delta-reporter/delta-frontend/pull/29/commits/b2ad461374d2e03b0558c66c21cb447302515e59#diff-74a2ed0155b7d85e0772eae9b0c5bf71L58
where is this paragraph  coming from?

File: pages/charts.tsx (hunk 1 above)
Comment (Jnegrier) [reply pages/charts.tsx:122]: hahahaha I really believed it was me, looks like something I would do. I fixed the test that was failing, so this is ready to go 🚀 
Thanks for adding the current tag on historical tests btw

**Comment by Jnegrier** [comment]
> @Jnegrier what are those tests actually doing?
> [b2ad461#diff-74a2ed0155b7d85e0772eae9b0c5bf71L58](https://github.com/delta-reporter/delta-frontend/commit/b2ad461374d2e03b0558c66c21cb447302515e59#diff-74a2ed0155b7d85e0772eae9b0c5bf71L58)
> where is this paragraph coming from?
> thanks for fixing the test

These tests are just taking screenshots of the app, and sending these screenshots to a service called Argos

File: pages/charts.tsx (hunk 1 above)
Comment (opishcheiko) [reply pages/charts.tsx:122]: it didn't like your fix either

Comment (opishcheiko) [reply pages/charts.tsx:122]: let's have a look tomorrow why it's not clickable

Comment (Jnegrier) [reply pages/charts.tsx:122]: Yep, its picky, I'm adding wait everywhere now 🔫

Comment (opishcheiko) [reply pages/charts.tsx:122]: 🔧

Comment (opishcheiko) [reply pages/charts.tsx:122]: your prayers worked! thanks, and good night

**Pull request merged**